### PR TITLE
fix: inconsistent use of init & extra containers on jobs

### PR DESCRIPTION
**Pull request by adamstrawson**
The Hydra helm-chart `job` resource has an incorrectly wrapped conditional, `extraContainers` is wrapped within `extraInitContainers` conditional, meaning you cant define an extra container without also setting `extraInitContainers`.

Keto & Kratos was also missing `extraInitContainers` entirely too. 

Included various other consistency issues across the three migration jobs. 

## Related Issue or Design Document

Issue: #507

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [x] I have referenced an issue containing the design document if my change introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got green light (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: helm/charts/keto/values.yaml [hunk 1]
```diff
@@ -54,6 +54,11 @@ job:
   #  - name: ...
   #    image: ...
 
+  extraInitContainers: {}
+  # extraInitContainers: |
+  #  - name: ...
+  #    image: ...
+
```
Comment (Demonsthere) [inline helm/charts/keto/values.yaml:61]:
```suggestion
  # -- If you want to add extra init containers.
  extraInitContainers: {}
  # extraInitContainers: |
  #  - name: ...
  #    image: ...
```

File: helm/charts/kratos/values.yaml [hunk 2]
```diff
@@ -428,6 +431,11 @@ job:
   #  - name: ...
   #    image: ...
 
+  extraInitContainers: {}
+  # extraInitContainers: |
+  #  - name: ...
+  #    image: ...
+
```
Comment (Demonsthere) [inline helm/charts/kratos/values.yaml:438]:
```suggestion
  # -- If you want to add extra init containers.
  extraInitContainers: {}
  # extraInitContainers: |
  #  - name: ...
  #    image: ...
```

**Review by Demonsthere — COMMENTED** [review]
Nice catch with the hydra job, looks like the work of a copy&paste error from some previous refactor 😓  I have updated the comments to be in line with our automation. Could you just add some test values to the `hack` for the CI?

**Comment by adamstrawson** [comment]
Thanks, I've updated `hack` with some test values. Hydra already had values in place, so just Kratos and Ketos to include. 

Appreciate the quick responses on these PR's 🙌

**Review by Demonsthere — APPROVED** [review]
LGTM 👌

**Pull request merged**